File: VRMeetingRoomCapsule/frontend/unity/Scripts/LangRouterClient.cs
using UnityEngine;

public class LangRouterClient : MonoBehaviour
{
    private string currentLang = "en";

    public string GetLang()
    {
        return currentLang;
    }

    public void SetLang(string lang)
    {
        currentLang = string.IsNullOrEmpty(lang) ? "en" : lang;
        Debug.Log("Language changed (placeholder): " + currentLang);
    }
}

File: VRMeetingRoomCapsule/frontend/webxr/src/LangRouterClient.ts
export class LangRouterClient {
  private currentLang = "en";

  getLang() {
    return this.currentLang;
  }

  setLang(lang: string) {
    this.currentLang = lang && lang.trim().length ? lang : "en";
  }

  async localize(key: string, fallback: string) {
    void key;
    return fallback;
  }
}

